src/dom/native/NativeViewElementNode.ts

```diff
@@ -50,7 +50,7 @@ export default class NativeViewElementNode<T extends View> extends NativeElement
         let oldAnimations: KeyframeAnimation[] = [];
 
         const addAnimation = (animation: string) => {
-            log.debug(`Adding animation ${animation}`)
+                log.debug(`Adding animation ${animation}`)
```

Comment:
Not sure this is wanted.

```suggestion
            log.debug(`Adding animation ${animation}`)
```

Same for lines below.